data/os/FreeBSD.yaml

```diff
@@ -19,7 +19,5 @@ mcollective::server_config:
   plugin.rpcaudit.logfile: "/var/log/mcollective-audit.log"
   factsource: "yaml"
   plugin.yaml: "/usr/local/etc/mcollective/generated-facts.yaml"
-  registerinterval: 10
```

Comment:
Is this what you meant in https://github.com/choria-io/puppet-choria/pull/189#discussion_r509535308 ?

Reply:
Also maybe here https://github.com/choria-io/puppet-mcollective/blob/master/data/common.yaml#L21-L22

Reply:
https://github.com/choria-io/mcorpc-ruby-support/pull/73 should also help

Reply:
We should also stop building server facts https://github.com/choria-io/puppet-mcollective/blob/8e0ea1e1b35ed798f6224feefc68fc6bc32c6310/lib/facter/mcollective.rb#L19 since its pointless

Reply:
awesome. since I removed it for freebsd, the puppet run and the cronjob work without issues. IMO the PR is ready. Maybe @smortex can review/test this as well?

Reply:
Morning!  I'll give it a shot today, but first I have to tidy out some WIP related to updating Puppet to 6.19.0 (and I found a problem thanks to reviewing this PR, so thanks :+1: !)

Reply:
Seems to be good!